p2p/monero-p2p/src/client.rs

```diff
@@ -29,13 +29,13 @@ pub use handshaker::{DoHandshakeRequest, HandShaker, HandshakeError};
 use monero_pruning::PruningSeed;
 
 /// An internal identifier for a given peer, will be their address if known
-/// or a random u64 if not.
+/// or a random u128 if not.
```

Comment:
Question: what's the reasoning for the switch here? Is `u64` not big enough to avoid random conflicts?

Reply:
Yeah, realistically `u64` is big enough to avoid collisions but moving moving to a `u128` makes it pretty much impossible.